.storybook/main.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
   "stories": [
-    "../src/**/*.stories.mdx",
-    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+    "../src/**/*/stories.@(ts|tsx)",
```

Comment:
now we can just name stories as `stories.tsx`, the prefix is redundant imo, as the file is in the directory anyways